mathjax3-ts/output/chtml/Wrappers/mrow.ts

```diff
@@ -49,16 +49,19 @@ export class CHTMLmrow extends CHTMLWrapper {
      * @override
      */
     public toCHTML(parent: HTMLElement) {
-        let chtml = parent;
+        let chtml = this.chtml = parent;
         if (!this.node.isInferred) {
             chtml = this.standardCHTMLnode(parent);
         }
         let hasNegative = false;
         for (const child of this.childNodes) {
             child.toCHTML(chtml);
-            if (child.bbox && child.bbox.w < 0) {
+            if (child.bbox.w < 0) {
```

Comment:
Is child.bbox guaranteed to exist?

Reply:
Yes, it is created in the Wrapper constructor at [line 266](https://github.com/mathjax/mathjax-v3/blob/master/mathjax3-ts/output/chtml/Wrapper.ts#L266)